src/module.c

```diff
@@ -801,7 +821,7 @@ int TSDB_createRule(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
 
 
 /*
-TS.INCRBY ts_key NUMBER [RESET time-bucket]
+TS.INCRBY ts_key NUMBER [TIMESTAMP timestamp] [RESET time-bucket]
```

Comment:
this shouldn't be here.

Reply:
Apparently I had some leak between PRs.
Once we accept the original PR these will be gone so let's wait with it. (it goes beyond these few lines).